javascript/javascript3/week3/shoping_cart.js

```diff
@@ -0,0 +1,134 @@
+console.log('Shoping cart');
+
+                /* Part 1 */
+
+     class Product {
+     constructor(name, price) {
+      this.name = name;
+      this.price = price;
+     }
+
+     async convertToCurrency(curr) {
+        const currency=curr.toUpperCase();
+        let price = 0;
+        const rice = await fetch("https://api.exchangeratesapi.io/latest?base=DKK")
+          .then((promise) => promise.json())
```

Comment:
since you're using async / await, try to avoid .then(), you should try not to mix them. Rewrite all of this function to await syntax and see how much better it is to work with.